lib/vagrant/plugin/v2/trigger.rb

```diff
@@ -51,27 +59,41 @@ def fire_triggers(action, stage, guest_name)
               guest_name: guest_name
           end
 
-          triggers = filter_triggers(triggers, guest_name)
+          triggers = filter_triggers(triggers, guest_name, type)
 
           if !triggers.empty?
             @logger.info("Firing trigger for action #{action} on guest #{guest_name}")
```

Comment:
This logger output needs to be updated to display the type of trigger being fired instead of the static `action` value (since `hook` is now valid and expected depending on context).